signal_scanner_bot/env.py

```diff
@@ -148,3 +154,14 @@ def _cast_to_path(to_cast: str) -> Path:
 SIGNAL_LOCK = Lock()
 STATE = _State(AUTOSCAN_STATE_FILE_PATH)
 TWITTER_TO_SIGNAL_QUEUE: Queue = Queue(maxsize=10000)  # shooting from the hip here...
+
+################################################################################
+# Peony Twitter Event Stream client
+################################################################################
+API_KEYS = {
+    "consumer_key": TWITTER_API_KEY,
+    "consumer_secret": TWITTER_API_SECRET,
+    "access_token": TWITTER_ACCESS_TOKEN,
+    "access_token_secret": TWITTER_TOKEN_SECRET,
+}
+CLIENT = peony.PeonyClient(**API_KEYS)
```

Comment:
Declaring a new public variable that is a Dict of the various API keys with the Dict keys formatted for how the peony client wants things to be. Then declare a public variable for a client. The client is (I think) going to just be used to do the actual tweet sending.